.github/workflow/todo-to-issue.yml

```diff
@@ -0,0 +1,15 @@
+# https://github.com/marketplace/actions/todo-to-issue
```

Comment:
can we call the directory `workflows`?
sorry for the nit

also dope workflow :)